Author: upstream (philippe)
Uopstream: committed in svn (r13471)
Reason: do not include linux/ptrace.h to avoid redefinitions.

--- a/coregrind/vgdb.c
+++ b/coregrind/vgdb.c
@@ -102,7 +102,6 @@
 #include <sys/user.h>
 #if defined(VGO_linux)
 #  include <sys/prctl.h>
-#  include <linux/ptrace.h>
 #endif
 #endif
 
